.github/workflows/build-storybook-main.yml

```diff
@@ -0,0 +1,55 @@
+name: Build Storybook Main
+
+on:
+  push:
+    branches:
+      - main
+  workflow_call:
+    secrets:
+      PUBLISH_STORYBOOK_TOKEN:
+        required: true
```

Comment:
It looks like this token is no longer being used because the `deploy-pages` action mints its own token. So maybe we can remove the secret requirement?

``` suggestion
```

Reply:
Yes! Thanks Elliot, @itsyoboieltr also pointed out I'm no longer using the secret and after further investigation it's not longer needed! [See comment](https://github.com/MetaMask/metamask-design-system/pull/610#discussion_r2072231028)